### ISSUE TITLE
new version breaks Contao\Controller::redirect

### ISSUE THREAD

**Issue by fritzmg** [terminal42/header-replay-bundle#2]
As mentioned in https://github.com/contao/core-bundle/issues/974, version `1.1.0` of `header-replay-bundle` of `Contao\Controller::redirect` when used outside of regular pages. For example if you have an `initializeSystem` hook which might call `Contao\Controller::redirect` the redirect does not work anymore when `terminal42/header-replay-bundle` is installed in version `1.1.0`.

If you have `terminal42/header-replay-bundle` installed in version `1.0.0` or `1.0.1` everything still works as expected.

To reproduce, just do
```php
\Controller::redirect('https://www.google.com/');
```
within an `initializeSystem` hook and then open the front end. The redirect won't happen when `header-replay-bundle` `1.1.0` is installed.

**Comment by Toflar** [comment]
I tried to reproduce but I can't. My hook correctly redirects. However, I think I understand where the problem **might** occur. Can you try if the hotfix branch (https://github.com/terminal42/header-replay-bundle/tree/hotfix/1.1.1) fixes your issue?

**Comment by fritzmg** [comment]
The redirect works with `hotfix/1.1.1`. However, the Contao Install Tool does not work. When I try to access it, the browser tries to download a file of the type `application/vnd.t42.header-replay`. Raw response header:
```
Date: Wed, 19 Jul 2017 11:46:00 GMT
Server: Apache/2.4.25 (Win32) OpenSSL/1.0.2j PHP/7.1.4
x-powered-by: PHP/7.1.4
Cache-Control: no-cache, private
X-Content-Type-Options: nosniff
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
x-frame-options: SAMEORIGIN
x-xss-protection: 1; mode=block
Content-Length: 0
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Content-Type: application/vnd.t42.header-replay
```

**Comment by Toflar** [comment]
I've pushed yet another commit :)

**Comment by fritzmg** [comment]
Looks good :). Frontend, redirect in `initializeSystem` hook and the Contao Install Tool seem to be working now.

**Comment by Toflar** [comment]
*phew* :) Released 1.1.1 then, thank you!
